### ISSUE TITLE
Cluster creation failing with weird characters

### ISSUE THREAD

**Issue by VidyasagarMSC** [IBM-Cloud/terraform-provider-ibm#556]
Hi there,

While applying terraform script to create a Kubernetes cluster, it's failing with weird characters in the output 

> ibm_container_cluster.cluster: ibm_container_cluster.cluster: Error communicating with the API: Request failed with status code: 503, ServerErrorResponse: ���
> �@���\�F��VEV�
>               ��$.��c��	�{	�n9��
>                                      �U��s��f��� !*K�7/�kv�r���q̬2���"<��@���ĉ�ݬ���a��#E�K�[����&Й�-F��{9��>`��Ќ5ǻP}y�O?�"!ה�_N�4`��{7�>_鮩� 
> 

### Terraform Version
0.11.3

Provider: 0.16.0

### Affected Resource(s)
ibm_container_cluster

### Expected Behavior
It should create a cluster or at least show the error message with valid characters

### Actual Behavior
Fails with improper characters

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. terraform init
2. terraform plan
3. Fails

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by vijaylakshmi-b** [comment]
Facing the same issue.

Terraform is unable to fetch the state of the cluster and times out instantly.
However, the cluster gets created successfully on the bluemix console.

### Terraform Version
Terraform v0.11.13

IBM Cloud Provider version 0.14.1

### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_container_cluster

### Terraform Configuration Files
```hcl
resource "ibm_container_cluster" "bfast-resource" {
  name = "${var.name}"
  datacenter = "${var.datacenter}"
  machine_type = "${var.machine_type}"
  hardware = "${var.hardware}"
  region="${var.region}"
  resource_group_id="${var.resource_group_id}"
}
```
### Debug Output
https://gist.github.com/vijaylakshmi-b/e5d6ad547835241380ef495c3f9cdf63

### Expected Behavior
Container status should be successfully returned and tfState file should be created

### Actual Behavior
terraform execution crashes with error
`Error waiting for cluster (feadba97be4746e9baa8a9b9657ec07b) to become ready: Error retrieving cluster: Request failed with status code: 503, ServerErrorResponse: ï¿½\00\00\00\00\00\00\00ï¿½ï¿½ï¿½
2019-04-19T11:04:18.525704632Z ï¿½@ï¿½ï¿½ï¿½ï¿½ï¿½\ï¿½Fï¿½ï¿½VEVï¿½ï¿½ï¿½$.ï¿½ï¿½cï¿½ï¿½	ï¿½ï¿½{	ï¿½n9ï¿½ï¿½ï¿½ï¿½Uï¿½ï¿½sï¿½ï¿½fï¿½ï¿½ï¿½ !*Kï¿½7/ï¿½kï¿½vï¿½rï¿½ï¿½ï¿½qÌ¬2ï¿½ï¿½ï¿½"<ï¿½ï¿½@ï¿½ï¿½ï¿½Äï¿½Ý¬ï¿½ï¿½ï¿½aï¿½ï¿½#Eï¿½Kï¿½[ï¿½ï¿½ï¿½ï¿½&Ðï¿½-Fï¿½ï¿½{9ï¿½ï¿½>`ï¿½ï¿½Ð5Ç»P}yï¿½O?ï¿½"!×ï¿½_Nï¿½4`ï¿½ï¿½{7ï¿½>_é®©ï¿½\00\00\00`

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by Praveengostu** [comment]
Hi @VidyasagarMSC @vijaylakshmi-b We had an similar issue earlier and fixed in release 0.15.1.. We will check this if it recreatable in v0.16.0 as mentioned by @VidyasagarMSC  and let you know the updates.

**Comment by Praveengostu** [comment]
Hi @VidyasagarMSC @vijaylakshmi-b I was able to successfully provision my cluster with provider v0.16.0. Could you please try with latest provider by enabling the debug log `export TF_LOG=debug` and share us the same if you are able to recreate the issue.
```
ibm_container_cluster.cluster: Still creating... (29m10s elapsed)
ibm_container_cluster.cluster: Still creating... (29m20s elapsed)
ibm_container_cluster.cluster: Still creating... (29m30s elapsed)
ibm_container_cluster.cluster: Still creating... (29m40s elapsed)
ibm_container_cluster.cluster: Still creating... (29m50s elapsed)
ibm_container_cluster.cluster: Still creating... (30m0s elapsed)
ibm_container_cluster.cluster: Still creating... (30m10s elapsed)
ibm_container_cluster.cluster: Still creating... (30m20s elapsed)
ibm_container_cluster.cluster: Still creating... (30m30s elapsed)
ibm_container_cluster.cluster: Still creating... (30m40s elapsed)
ibm_container_cluster.cluster: Still creating... (30m50s elapsed)
ibm_container_cluster.cluster: Still creating... (31m0s elapsed)
ibm_container_cluster.cluster: Creation complete after 31m3s (ID: )

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

**Comment by VidyasagarMSC** [comment]
@Praveengostu Tried an hour ago and seeing the same error. Also, Why is IBM Cloud provider version showing as unversioned?
<img width="714" alt="Screenshot 2019-04-22 at 11 12 07 AM" src="https://user-images.githubusercontent.com/6552440/56484963-9f0e8100-64ef-11e9-86d1-01227141bae9.png">

**Comment by VidyasagarMSC** [comment]
Updating the terraform provider path in `.terraformrc` worked for me

**Comment by Praveengostu** [comment]
@vijaylakshmi-b terraform provider version you can check by running the `terraform init`. If the binary name is changed it should display the version of the provider which you are using when running the terraform commands. Pls close this issue if your issue is fixed with v0.16.0
